src/sign.builds

```diff
@@ -32,12 +32,14 @@
     <ReadSigningRequired MarkerFiles="@(SignMarkerFile)">
       <Output TaskParameter="SigningMetadata" ItemName="FilesToSign" />
     </ReadSigningRequired>
+    <Message Text="Attempting to sign %(FilesToSign.Identity) with authenticode %(FilesToSign.Authenticode) and strongname %(FilesToSign.StrongName)" />
```

Comment:
This didn't show up in the build logs. We should probably change the importance to High to have it show up. Can you do that when you remove the ContinueOnError below?